src/renderer/components/outputs/PyTorchOutput.tsx

```diff
@@ -63,6 +72,14 @@ export const PyTorchOutput = memo(
                                 'outputChannels',
                                 new NumericLiteralType(value.outNc)
                             ),
+                            new NamedExpressionField(
+                                'modelType',
+                                new StringLiteralType(convertToInternalModelType(value.modelType))
```

Comment:
This seems weird. Why do you map `RealESRGAN` to `ESRGAN`?

Reply:
Because those model types are compatible. We talked about this in discord. Internally we convert RealESRGAN models to esrgan models

Reply:
Okay, then why does the backend send "RealESRGAN" then?

Reply:
Because it starts as a RealESRGAN type model and mainly people think of RealESRGAN as something different (which I guess technically it is even though it can be converted to the same format). 

Though even though technically they can be interpolated with regular ESRGAN models, I don't think any exist which would actually work.

Reply:
> it can be converted to the same format

Where does this conversion happen?

Reply:
Inside RRDB.py

Reply:
So models get converted on load. This means that the `modelType` field is ill-defined right now. Does model type represent the original (unconverted) model type or the actual (converted) mode type?

(Also, the mapping "RealESRGAN" -> "ESRGAN" is not enough, because we also have "ESRGAN+". You probably don't need to add this mapping though. My end goal is to get rid of the need to do this mapping altogether.)

Reply:
ESRGAN+ models are not compatible with ESRGAN or RealESRGAN models, and RealESRGAN models cannot be +

Anyway, it represents the unconverted model. I suppose it should probably represent the converted model though

Reply:
> I suppose it should probably represent the converted model though

Sounds good.